apps/docs/docusaurus.config.js

```diff
@@ -24,7 +24,7 @@ const config = {
         docs: {
           sidebarPath: require.resolve('./sidebars.js'),
           routeBasePath: '/',
-          editUrl: '/',
+          editUrl: 'https://github.com/origranot/reduced.to',
```

Comment:
Hey @barkhaaroraa,
editUrl is wrong, take a look at: https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-content-docs